src/app/tests/suites/certification/Test_TC_DD_2_1.yaml

```diff
@@ -123,8 +123,9 @@ tests:
           Try NRF Connect app (https://www.nordicsemi.com/Products/Development-tools/nrf-connect-for-desktop)
           OR
           HCIDump (https://ubuntu.com/core/docs/bluez/reference/commands)
-          ->For T0 and 30s we have to get advertisement range between 20ms to 60ms
-          ->For 30s and 15mins we have to get advertisement range between 150ms to 1200ms
+          ->Between [0 .. 30s[ expect advertisement range between 20ms to 60ms.
```

Comment:
Merge gate comment: resolve only after https://github.com/CHIP-Specifications/chip-test-plans/pull/4020 is merged.

Reply:
Spec PR has been merged, with broad review / approval

Reply:
Also merged